dev-libs/atkmm.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import stdlib
+from stdlib.template import autotools
+from stdlib.manifest import manifest
+
+
+@manifest(
+    name='atkmm',
+    category='dev-libs',
+    description='''
+    C++ bindings for ATK.
+    ''',
+    tags=['atk', 'c++', 'bindings'],
```

Comment:
```suggestion
    tags=['atk', 'cpp', 'bindings'],
```

(Am tags don't accept the `+` symbol. Yes we should change that, but now is not the time).